bin/main.rb

```diff
@@ -1,35 +1,45 @@
-# Set up game
-puts "Welcome!"
-puts "Player 1, What is your name?"
-@player1 = gets.chomp
-puts "Now, Player 2, What is your name?"
-@player2 = gets.chomp
-puts "Let's play! #{@player1} #{@player2}"
-count = 0
+require_relative  '../lib/game.rb'
+puts
+puts "*" * 27
+puts "  WELCOME TO TIC-TAC-TOE!"
+puts "*" * 27
+puts
+print "Player 1, What is your name?: "
+player1 = gets.chomp.capitalize
+print "Now, Player 2, What is your name?: "
+player2 = gets.chomp.capitalize
+puts
+puts "Let's play! #{player1} and #{player2}!!"
+puts
```

Comment:
[Optional]It will be better to put this in a function like def welcome_msg

Reply:
Oh wow, I didn't realize that having the same name would cause such a weird bug! Thank you for pointing these out @ispirett! We will look into those immediately! :)

Reply:
In pointing out the bug where a winner should have been displayed, you actually found a bug that we hadn't noticed before.

It wasn't that the program wasn't declaring a winner (because it did when I played those exact same patterns) but that the game was accepting inputs other than integers! 

Thank you for finding this. :)

Reply:
Yeah, I forgot to mention that I could have put input wierd stuff.

Reply:
Really?? I didn't know you could do that! I can add that to something I learned today. Thanks @Ispirett!